src/dms/frontend/EdFi.DataManagementService.Frontend.AspNetCore/Modules/TokenEndpointModule.cs

```diff
@@ -14,11 +17,28 @@ public void MapEndpoints(IEndpointRouteBuilder endpoints)
         endpoints.MapPost("/oauth/token", GenerateToken);
     }
 
-    internal static async Task GenerateToken(HttpContext httpContext)
+    internal static async Task GenerateToken(HttpContext httpContext, IOptions<AppSettings> appSettings, IOAuthManager oAuthManager, ILogger<TokenEndpointModule> logger, IHttpClientFactory httpClientFactory)
     {
-        var tokenDetails = new TokenResponse("temporary-fake-token", 1800, "bearer");
-        httpContext.Response.StatusCode = StatusCodes.Status200OK;
-        await httpContext.Response.WriteAsSerializedJsonAsync(tokenDetails);
+        // Create client for sending upstream request.
+        logger.LogInformation("Handling OAuth Token request.");
```

Comment:
The following will be useful. Will require changing `ExtractTraceIdFrom` from private to public.

```csharp
        var traceId = AspNetCoreFrontend.ExtractTraceIdFrom(httpContext.Request, appSettings);
        logger.LogInformation("Received token request for forwarding to identity provider - {TraceId}", traceId);
```